aspnetcore/fundamentals/openapi/buildtime-openapi.md

```diff
@@ -36,57 +37,85 @@ Run the following command in the directory that contains the project file:
 ```dotnetcli
 dotnet add package Microsoft.Extensions.ApiDescription.Server --prerelease
 ```
+
 ---
 
-Upon installation, this package will automatically generate the Open API document(s) associated with the application during build and populate them into the application's output directory.
+The `Microsoft.Extensions.ApiDescription.Server` package automatically generates the Open API document(s) associated with the app during build and populates them into the app's output directory:
+
+Consider a template created API app named `MyTestApi`:
+
+### [Visual Studio](#tab/visual-studio)
+
+The Output tab in Visual Studio includes information similar to the following:
+
+```text
+1>Generating document named 'v1'.
+1>Writing document named 'v1' to 'MyProjectPath\obj\MyTestApi.json'.
+```
+
+### [.NET CLI](#tab/net-cli)
+
+The following commands build the app and display the generated OpenAPI document:
 
 ```cli
 $ dotnet build
-$ cat bin/Debub/net9.0/{ProjectName}.json
+$ cat obj/MyTestApi.json
 ```
 
-## Customizing build-time document generation
+---
+
+The generated `obj/{MyProjectName}.json` file contains the [OpenAPI version, title,  endpoints, and more](https://learn.openapis.org/specification/structure.html). The first few lines of obj/MyTestApi.json file:
 
-### Modifying the output directory of the generated Open API file
+:::code language="json" source="~/fundamentals/openapi/samples/9.x/BuildTime/csproj/MyTestApi.json" id="snippet_1" highlight="4-5":::
 
-By default, the generated OpenAPI document will be emitted to the application's output directory. To modify the location of the emitted file, set the target path in the `OpenApiDocumentsDirectory` property.
+## Customize build-time document generation
 
-```xml
-<PropertyGroup>
-  <OpenApiDocumentsDirectory>./</OpenApiDocumentsDirectory>
```

Comment:
`./` generates misleading error: Missing required option '--project'.
I replaced it with a single dot `.`

Reply:
Interesting -- I had success with this optional locally. Out of curiosity, did you try this on the BuildTime sample with the RC1 package versions?

Reply:
> Interesting -- I had success with this optional locally. Out of curiosity, did you try this on the BuildTime sample with the RC1 package versions?

Yes, using todays build RC1 `9.0.100-rc.1.24415.1`

Reply:
In what directory do you run the `dotnet build` when this error occurs?

Reply:
I'm running it in the project folder. If I remove that  element or change it to `..` the build succeeds.

Reply:
Yeah, I am still not able to repro. I suspect the issue might be OS-dependent since I am on a Mac?

I'll try to run it on Windows and see if the issue repros.

Reply:
OK, I was able to repro this on Windows. It looks like it's a path formatting related issue. Let's stick to using:

```
<OpenApiDocumentsDirectory>.</OpenApiDocumentsDirectory>
```

in the docs for now.

Reply:
> Yeah, I am still not able to repro. I suspect the issue might be OS-dependent since I am on a Mac?
> 
> I'll try to run it on Windows and see if the issue repros.

@martincostello  do you have access to windows? If so, can your verify?

Reply:
@captainsafia That's it! On windows `.` and `.\' work.